app/core/config.py

```
@@ -0,0 +1,293 @@
+    
+    Manages JWT tokens, CORS settings, and application secrets.
+    """
+    
+    secret_key: SecretStr = Field(..., description="Application secret key")
+    jwt_algorithm: str = Field(default="HS256", description="JWT signing algorithm")
+    jwt_expiration_minutes: int = Field(
+        default=30,
+        ge=1,
+        description="JWT token expiration time in minutes"
+    )
+    allowed_origins: List[str] = Field(
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)